adafruit_platformdetect/board.py

```diff
@@ -50,6 +50,7 @@ class Board:
 
     def __init__(self, detector):
         self.detector = detector
+        self.board_asset_tag = self.detector.check_board_asset_tag_value().strip()
```

Comment:
Let's add the strip() into the check_board_asset_tag_value() function.

Reply:
Done.